ui/components/app/edit-gas-fee-popover/edit-gas-fee-popover.js

```diff
@@ -40,17 +43,23 @@ const EditGasFeePopover = () => {
                 <I18nValue messageKey="gasOption" />
               </span>
               <span className="edit-gas-fee-popover__content__header-time">
-                <I18nValue messageKey="time" />
+                {editGasMode !== EDIT_GAS_MODES.SWAPS && (
+                  <I18nValue messageKey="time" />
+                )}
```

Comment:
just curious why don't we include time estimations for swaps?

Reply:
by designs, need to check with Jake